charts/rekor-tiles/rekor-tiles/templates/deployment.yaml

```diff
@@ -0,0 +1,108 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
```

Comment:
we should support specifying annotations on the deployment

Reply:
done